index.js

```diff
@@ -360,6 +360,32 @@ exports.ecsign = function (msgHash, privateKey) {
 }
 
 /**
+ * Returns the keccak-256 hash of `message`, prefixed with the header used by the `eth_sign` RPC call.
+ * The output of this function can be fed into `ecsign` to produce the same signature as the `eth_sign`
+ * call for a given `message`, or fed to `ecrecover` along with a signature to recover the public key
+ * used to produce the signature.
+ * @param message
+ * @returns {Buffer} hash
+ */
+exports.hashForEthSign = function (message) {
```

Comment:
I think this should be named differently, given it is not for transaction signing. Perhaps something like `hashPersonalMessage` because it initially was created as the `personal.sign` RPC method.